IBX-8916: Updates in User documentation (#313)
ibexa/documentation-user: docs/customer_management/customer_management.md

---

> description: Customer management allows you to manage and view accounts for customers and organizations in your system, as well as create customer portals and allow for company self-registration.

Reviewer: ```suggestion
description: Use customer management features to manage and view accounts for customers and organizations in your system, as well as create customer portals and allow for company self-registration.
```

---

# Customer management

> Customer management allows you to manage and view accounts for customers and organizations in your system, as well as create customer portals and allow for company self-registration.

Reviewer 2: ```suggestion
Customer management allows you to manage and view accounts for customers and organizations in your system, as well as create customer portals and allows for company self-registration.
```

> Customer management allows you to manage and view accounts for customers and organizations in your system, as well as create customer portals and allow for company self-registration.

Reviewer: ```suggestion
By using customer management features you can manage and view accounts for customers and organizations in your system, as well as create customer portals and allow for company self-registration.
```

Author: I'd keep first version.
